src/2018/transitioning/ownership-and-lifetimes/anonymous-lifetime.md

```diff
@@ -31,9 +33,11 @@ impl<'a> fmt::Debug for StrWrap<'a> {
 }
 ```
 
-In edition 2018, you can instead write:
+In Rust 2018, you can instead write:
 
 ```rust
+// Rust 2018
+
 #![feature(rust_2018_preview)]
```

Comment:
The comment and the feature gate need to be swapped lexically to make this pass the test, at least for now.

Reply:
This is https://github.com/rust-lang/rust/issues/51685